src/main/java/io/openliberty/website/SecurityFilter.java

```diff
@@ -71,7 +71,7 @@ public void doFilter(ServletRequest req, ServletResponse resp, FilterChain chain
             response.setHeader("X-Content-Type-Options", "nosniff");
              // Mitigating cross site scripting (XSS) from other domains.
             response.setHeader("Content-Security-Policy",
-                    "default-src 'self' 'unsafe-inline' 'unsafe-eval' cdn.jsdelivr.net fonts.googleapis.com ajax.googleapis.com code.jquery.com fonts.gstatic.com  *.githubusercontent.com api.github.com www.googletagmanager.com tagmanager.google.com www.google-analytics.com cdnjs.cloudflare.com data: buttons.github.io www.youtube.com video.ibm.com https://start.openliberty.io/ gitlab.com starter-staging.rh9j6zz75er.us-east.codeengine.appdomain.cloud https://docs.oracle.com/javase/8/docs/api/ https://docs.oracle.com/en/java/javase/17/docs/api/");
+                    "default-src 'self' 'unsafe-inline' 'unsafe-eval' cdn.jsdelivr.net ajax.googleapis.com code.jquery.com fonts.gstatic.com  *.githubusercontent.com api.github.com www.googletagmanager.com tagmanager.google.com www.google-analytics.com cdnjs.cloudflare.com data: buttons.github.io www.youtube.com video.ibm.com https://start.openliberty.io/ gitlab.com starter-staging.rh9j6zz75er.us-east.codeengine.appdomain.cloud https://docs.oracle.com/javase/8/docs/api/ https://docs.oracle.com/en/java/javase/17/docs/api/");
```

Comment:
there is a reference to ajax.googleapis.com , pls check if that is allowed or not as well.

Reply:
Since the issue specifies only google web fonts, I think we are good to go with this for now. I have asked @NottyCode regarding the same.